var/spack/repos/builtin/packages/sensei/package.py

```diff
@@ -69,39 +69,38 @@ def cmake_args(self):
             '-DCMAKE_C_FLAGS=-fPIC -Wall -Wextra -O3 -mtune=generic'
         ]
 
-        if (spec.variants['catalyst'].value):
+        if ('catalyst' in spec)
```

Comment:
while both of these evaluate the same, `'variantname' in spec` is a nice syntactic sugar that was implemented and is generally the way most spack packages are written.